tools/mo/unit_tests/mo/ops/select_test.py

```diff
@@ -292,11 +292,11 @@ def test_select_infer_tf_condition_assert_raises(self):
                                                       out_value=None, out_shape=shape_array([100, 20]),
                                                       auto_broadcast='numpy', fw_format='tf')
 
-    def test_select_infer_assert_pdpd(self):
-        with self.assertRaisesRegex(Error, "PDPD broadcasting rule is not implemented yet"):
+    def test_select_infer_assert_paddle(self):
+        with self.assertRaisesRegex(Error, "Paddle broadcasting rule is not implemented yet"):
             self.build_select_graph_and_infer(condition_value=None, condition_shape=[2, 2],
                                               then_value=None, then_shape=[2, 2],
                                               else_value=None, else_shape=[3, 3],
                                               out_value=None, out_shape=[42, 42],
-                                              auto_broadcast='pdpd')
+                                              auto_broadcast='paddle')
```

Comment:
incorrect. Please, perform self-review first
Looks like global find and replace did some incorrect replacements

Reply:
I thought it is Draft, sorry for inconvenience

Reply:
fixed